atlp-rwanda/elite-bn-frontend
# Ch react setup

**Description**

     This set up a project from scratch without using npx create-react-app

**Description of Task to be completed**

            -Install and configure Wepack 5 & Babel 7
            -Configured The App to Use TailwindCss
            -Set up React Router and created a dummy route "/" for the landing page.
            -Added a ReadMe file which includes: Installation and run commands

**How should this be manually tested?**

            -Install Node js or make sure You have it already
            -To clone, run https://github.com/atlp-rwanda/elite-bn-frontend.git
            -cd into the project directory
            -Run npm install in the terminal
            -To run the application in development mode, run npm run dev
            -Then open http://localhost:8080/ to view it in the browser.

**What are the relevant Trello story?**

Ch-react-setup >> https://trello.com/c/SI7Gb2nA

**What are Relevant Pictures**

<img width="1440" alt="Screen Shot 2021-01-19 at 13 56 45" src="https://user-images.githubusercontent.com/57200088/105031614-39517f00-5a5e-11eb-8cab-e276ec33414c.png">
